vshard.go

```diff
@@ -28,6 +28,7 @@ type Router struct {
 	}
 
 	knownBucketCount atomic.Int32
+	refID            atomic.Int64
```

Comment:
please write a small comment for this field

Reply:
u can use it from lua code https://github.com/tarantool/vshard/blob/master/vshard/router/init.lua#L60